log/logger.go

```diff
@@ -82,7 +88,7 @@ func redactKeyvals(keyvals ...interface{}) []interface{} {
 
 func RedactURL(str string) string {
 	strLower := strings.ToLower(str)
-	if !strings.HasPrefix(strLower, "s3+http") && !strings.HasPrefix(strLower, "http") {
+	if !strings.HasPrefix(strLower, "s3+http") && !strings.HasPrefix(strLower, "http") && !strings.HasPrefix(strLower, "s3") {
```

Comment:
Can we get rid of the first clause now you've added the more general one?

Reply:
oh yeah ha :)